tests/Kernel/fixtures/rendering.yml

```diff
@@ -388,6 +388,27 @@
       'article.ecl-content-item div.ecl-content-item__meta': "Meta 1 | Meta 2"
       'article.ecl-content-item div.ecl-content-item__title .ecl-link': "Item title"
       'article.ecl-content-item div.ecl-content-item__description': "Item text"
+- array:
```

Comment:
Any way we can add coverage also for non string cases? Because as you can see, the lack of it caused bugs in the actual trimming function..

Reply:
I added three more cases to check for the different render arrays.